From 7c2bff201e7b6cba8300658114667dd453c29f02 Mon Sep 17 00:00:00 2001
From: Paolo Bonzini <pbonzini@redhat.com>
Date: Fri, 18 Jul 2025 18:03:50 +0200
Subject: [PATCH 094/100] redhat: enable CONFIG_TDX

Signed-off-by: Paolo Bonzini <pbonzini@redhat.com>
---
 configs/devices/x86_64-softmmu/x86_64-rh-devices.mak | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/devices/x86_64-softmmu/x86_64-rh-devices.mak b/configs/devices/x86_64-softmmu/x86_64-rh-devices.mak
index 828cb8aa6f..c8dac4d083 100644
--- a/configs/devices/x86_64-softmmu/x86_64-rh-devices.mak
+++ b/configs/devices/x86_64-softmmu/x86_64-rh-devices.mak
@@ -74,6 +74,7 @@ CONFIG_SERIAL_PCI=y
 CONFIG_SEV=y
 CONFIG_SMBIOS=y
 CONFIG_SMBUS_EEPROM=y
+CONFIG_TDX=y
 CONFIG_TEST_DEVICES=y
 CONFIG_USB=y
 CONFIG_USB_EHCI=y
-- 
2.47.3

